docs/theme/mlflow/static/css/theme.css

```diff
@@ -4648,6 +4647,13 @@ div[class^='highlight'] pre {
     background: #343131;
     z-index: 8
 }
+.wy-nav-side.fixed {
+  position: fixed;
+  top: 0;
+}
+.wy-nav-side.relative {
+  position: relative;
```

Comment:
I need to add a relative mode in order to have the z-index take effect. If position is unset, the sidebar will be behind an element which is intended to push the main page over to the right, and this causes the links to be unclickable.